library/std/src/fs/tests.rs

```diff
@@ -40,6 +40,7 @@ macro_rules! error {
             Ok(_) => panic!("Unexpected success. Should've been: {:?}", $s),
             Err(ref err) => assert!(
                 err.raw_os_error() == Some($s),
+                "{}",
                 format!("`{}` did not have a code of `{}`", err, $s)
             ),
```

Comment:
No need to use format!() to format the argument to a format string. So instead of adding "{}", just remove the `format!()` macro:

```suggestion
            Err(ref err) => assert!(err.raw_os_error() == Some($s), "`{}` did not have a code of `{}`", err, $s),
```